tests/api/core/content-manager/api/number-of-draft-relations.test.api.js

```diff
@@ -28,31 +28,61 @@ const productModel = {
     name: {
       type: 'string',
       required: true,
+      pluginOptions: {
```

Comment:
I'm lacking some context here, do we have to do this because relations aren't localized by default so you have to manually specify in the schema?

Reply:
Yep, for this test I wanted them to be localized so when I started testing
across other locales the values weren't synced back to the default locale